backend/app/views/spree/admin/products/new.html.erb

```diff
@@ -6,60 +6,73 @@
 
     <legend align="center"><%= Spree.t(:new_product) %></legend>
 
-    <%= f.field_container :name do %>
-      <%= f.label :name, class: 'required' %><br />
-      <%= f.text_field :name, :class => 'fullwidth title', :required => true %>
-      <%= f.error_message_on :name %>
-    <% end %>
+    <div class="row">
+      <div class="col-lg-8 col-xs-12">
+        <div class="row">
+          <div class="col-xs-12">
+            <div class="row">
+              <% unless @product.has_variants? %>
+                <div data-hook="new_product_sku" class="col-md-6 col-xs-12">
```

Comment:
The problem is with holding less than full-width columns on a row by themselves.
Here we want the SKU field to always be on it's own line visually, but if the SKU div and the Name div are in the same `row`, they will sit on the same line (at medium and above).

Some better spacing here could make the grid structure read clearer.